### PR TITLE
feat: brand upgrade from "BitKeep" to "Bitget" in injected wallets

### PR DESCRIPTION
**Pull request by xusiyuan841028**
### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [x] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [x] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

#### Tests with demo app (SDK)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

### CODE REVIEW

File: packages/injected/src/index.ts [hunk 1]
```diff
@@ -41,6 +41,8 @@ function injected(options?: InjectedWalletOptions): WalletInit {
       ({ label }) => label
     )
 
+    debugger
```
Comment (Adamj1232) [inline packages/injected/src/index.ts:44]: please remove

File: packages/injected/src/types.ts [hunk 2]
```diff
@@ -172,7 +172,7 @@ export enum InjectedNameSpace {
   Arbitrum = 'arbitrum',
   XFI = 'xfi',
   GameStop = 'gamestop',
-  BitKeep = 'bitkeep',
+  Bitget = 'bitkeep',
```
Comment (Adamj1232) [inline packages/injected/src/types.ts:175]: does `'bitkeep'` need to be updated?

File: .gitignore [hunk 3]
```diff
@@ -5,4 +5,5 @@ package-lock.json
 .rpt2_cache
 .vscode
 yarn-error.log
-.env
\ No newline at end of file
+.env
+yarn.lock
```
Comment (Adamj1232) [inline .gitignore:9]: this can be removed

File: packages/injected/src/types.ts (hunk 2 above)
Comment (xusiyuan841028) [reply packages/injected/src/types.ts:175]: @Adamj1232 no, the `'bitkeep'` is the property key of `windows`, which is not changed.

File: .gitignore (hunk 3 above)
Comment (xusiyuan841028) [reply .gitignore]: @Adamj1232 updated by reverting the `.gitignore` file

File: packages/injected/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.10.8",
+  "version": "2.10.9",
```
Comment (Adamj1232) [inline packages/injected/package.json:3]:
```suggestion
  "version": "2.10.9-alpha.1",
```

**Review by Adamj1232 — APPROVED** [review]
Looks good, thanks for contributing!

**Pull request merged**